app/Importer/LicenseImporter.php

```diff
@@ -73,12 +73,21 @@ public function createLicenseIfNotExists(array $row)
                 $asset = Asset::where('asset_tag', $asset_tag)->first();
                 $targetLicense = $license->freeSeat();
                 if ($checkout_target) {
-                    $targetLicense->assigned_to = $checkout_target->id;
-                    $targetLicense->user_id = Auth::id();
-                    if ($asset) {
-                        $targetLicense->asset_id = $asset->id;
+                    try{
+                        $targetLicense->assigned_to = $checkout_target->id;
```

Comment:
This feels super-weird that we don't have some kind of License or LicenseSeat method to permit something to "be checked out" - fiddling with so many attributes by hand feels like it could easily go terribly wrong if we're not careful. If we don't have such a method, then So Be It, but if we do have something more object-oriented that we could use it would make me feel better.

Reply:
This is interesting... I just started making changes in the importer that already has code, but the code is, I think, a duplicate of the method `checkoutToAsset` from `LicenseCheckoutController.php`. you think that maybe I should use this method instead?